doc/remote.md

```diff
@@ -0,0 +1,37 @@
+## Run Lightning Terminal with a remote LND instance
```

Comment:
Why have two docs on configuring the remote mode? Shouldn't this content just live in the `config-remote-lnd.md` file?

Reply:
So yes and no. I had originally leaned towards one combined file but then realized that for the happy path a user would only need a subset of the information. Perhaps later we trim and merge the two?

Reply:
Ok combined into one and added the quickstart section